lib/mayaUsd/ufe/private/Utils.cpp

```diff
@@ -154,9 +154,13 @@ void applyCommandRestriction(const UsdPrim& prim, const std::string& commandName
         // if we have more than 2 layers that contributes to the final composed prim
         if (layers.size() > 1) {
             std::string layerDisplayNames;
-            for (auto layer : layers) {
-                layerDisplayNames.append("[" + layer->GetDisplayName() + "]" + ",");
-            }
+
+            // skip the the first arc which is PcpArcTypeRoot
+            // we are interested in all the arcs after root
+            std::for_each(std::next(layers.begin()),layers.end(), [&](const auto& it) {
+                 layerDisplayNames.append("[" + it->GetDisplayName() + "]" + ",");
+            });
+
             layerDisplayNames.pop_back();
```

Comment:
We previously were showing the first contributing layer which is of type Pcp.ArcTypeRoot a long with other contributing comp arcs. This was a bit confusing when I showed it to Natalia and it makes sense to skip it.

For example, in case of `/Kitchen_set/Arch_grp/Kitchen_1`, we would get following contributing layers in strength order:

```
[Kitchen.usd],
[Kitchen_payload.usd],
[Kitchen.geom.usd]
```

Reply:
Why is it confusing?  It's the correct information, no?  E.g. in the test/lib/usd/utils/test_CompositionQueryArc.cpp test you added, you specifically expect and assert that this arc is there.

Reply:
Hey @ppt-adsk , this change only affects the error report message. Using Kitchen_1 as an example, here what the message currently reads:

`Cannot rename [Kitchen_1] with definitions or opinions on other layers. Opinions exist in [Kitchen_set.usd],[Kitchen.usd],[Kitchen_payload.usd],[Kitchen.geom.usd]. `

The first arc is Pcp.ArcTypeRoot which points at [Kitchen_set.usd]. However since the message currently mentions "other layers", the error report could cause a bit of confusion. Therefore just displaying other layers in the error report:

`Cannot rename [Kitchen_1] with definitions or opinions on other layers. Opinions exist in [Kitchen.usd],[Kitchen_payload.usd],[Kitchen.geom.usd]. `

Reply:
Ah, got it, sounds great.  Can you turn the above GitHub comment into a code comment?